src/test/java/com/google/edith/UserInsightsTest.java

```diff
@@ -155,67 +155,69 @@ public void aggregateData() {
     newEntity4.setProperty("date", "2020-07-12");
```

Comment:
I think this test is testing two things at once. 1) Two entities with quantity 1 and 2 respectively and (2) entities with more than 1/2 quantities. Ideally, unit tests should be as isolated as possible and test one individual code path at a time (think one assert statement per test in an ideal world). Since this test is kinda large can we split up into two

Reply:
Okay, I separated it into two parts.